build.gradle

```diff
@@ -71,6 +72,14 @@ subprojects {
       prefix = "com.squareup.leakcanary"
       suppress = true
     }
+    packageOptions {
+      prefix = "shark.internal"
+      suppress = true
+    }
+    packageOptions {
+      prefix = "leakcanary.internal"
```

Comment:
Note: I haven't included those because I wanted to make sure there was nothing public in the internal package. Unfortunately I just changed that to share a HHPC data structure between two modules, so thanks!